### ISSUE TITLE
Svelte 5 errors

### ISSUE THREAD

**Issue by WoutPitje** [svar-widgets/gantt#1]
node_modules/wx-svelte-menu/node_modules/wx-svelte-core/src/components/Button.svelte:12:23 Declaring or accessing a prop starting with `$$` is illegal (they are reserved for Svelte internals) [plugin vite-plugin-svelte:optimize-svelte]

**Comment by JMaherBiscit** [comment]
Just made a PR for this if you wanna check it out https://github.com/svar-widgets/gantt/pull/2

**Comment by mkozhukh** [comment]
Yes, we are aware, and we are updating core libs now with the plan to fix the Gantt as well later
The Svelte 5 compatible version will be available till the end of the November

**Comment by mkozhukh** [comment]
Yep, having it Svelte 5 compatible is one thing, but making it SvelteKit compatible is a bit different.
Update to wx-svelte-gantt@2.0.0-beta-3 — it will work with SvelteKit.

**Comment by Zwillinge** [comment]
Starting from version  `v2.0.1` (released in the end of December 2024) svelte-gantt widget can be used with Svelte5 and SvelteKit
